openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCard/FeedCardHeader/FeedCardHeader.tsx

```diff
@@ -112,6 +112,22 @@ const FeedCardHeader: FC<FeedHeaderProp> = ({
     }
   };
 
+  const getAnnouncementLinkElement = () => {
```

Comment:
We will have function only if we will be doing some logic based on args else just have it as const

Reply:
Yes, In my next PR, I will update this as well as the other two functions to just const.